contrib/packaging/deb/cfg/debian/control

```diff
@@ -0,0 +1,19 @@
+Source: cilium
+Maintainer: Andre Martins <andre@cilium.io>
+Section: net
+Priority: optional
+Standards-Version: ${VERSION}
+Homepage: https://github/cilium/cilium
+Vcs-Git: git://github/cilium/cilium.git
+
+Package: cilium
+Architecture: amd64
+Section: net
+Priority: optional
+Depends: docker-engine (>=1.12), clang-3.8, libc6-dev-i386, iproute2 (>=4.6.0),
+ linux-image-4.8.1-040801-generic
```

Comment:
There's no difference. In my installation tests I've been using [this](http://kernel.ubuntu.com/~kernel-ppa/mainline/v4.8.1/) page to install the kernel since it's not available yet in the testing repos.